action/run-in-renode.py

```diff
@@ -530,6 +165,7 @@ def setup_renode():
         run_cmd(child, "#", 'echo "nameserver 1.1.1.1" >> /etc/resolv.conf')  # adds dns server address
 
         child.expect_exact("#")
+        child.logfile_read = None
```

Comment:
Add a comment why

Reply:
I think it is not necessary, I can delete that